### DISCUSSION TITLE
After erasing cookies all tabs and panels gone. After opening snapshot window all tabs in first panel or none

### DISCUSSION THREAD

**Discussion by wasntmewhodidit** [mbnuqw/sidebery#1828] [Q&A]
Hi there. Made snapshot, synchronized in Sideberry and Firefox mint-001 - 1.0 131.0 64bit. After that, all gone. Opened snapshots, chose the latest, opened window. All tabs in first panel. Tried again, tabs in no panel at all. Tried again, Tabs in first panel. And so on. At the moment, the 1000+ tabs are all open, but in no panel. I do not much about computers, we only get along as long as they do what they are supposed to. But I do not see that there is anything I could have done wrong. Could not find similar discussion. Any ideas would be appreciated.

**Comment by wasntmewhodidit** [comment]
Well then... this is the third time this happened... a lot of work again down the drain. No support, but thankfully lots of alternatives out there...

**Comment by jackysee** [comment]
Happened to me twice. All tabs moved to a single panel. I'm on Firefox nightly so I think it might be related?
